docs/site/Community-extensions.md

```diff
@@ -0,0 +1,92 @@
+[npmjs.org](https://www.npmjs.com/search?q=loopback4%20extension) for a complete
+list.
+
+{% include warning.html content="The extensions listed here are not supported by the LoopBack team; they are maintained by the LoopBack community and are listed here for convenience."%}
+
+## General
+
+<table>
+  <thead>
+    <tr>
+      <th width="160">Extension</th>
+      <th width="280">Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><a href="https://www.npmjs.com/package/loopback4-soft-delete">loopback4-soft-delete</a></td>
+      <td>Used for soft-deletes in a multi-tenant application</td>
+    </tr>
+    <tr>
+      <td><a href="https://www.npmjs.com/package/loopback4-helmet">loopback4-helmet</a></td>
+      <td>Integrate <a href="https://helmetjs.github.io/">helmetjs</a> in LoopBack applications</td>
+    </tr>
+    <tr>
+      <td><a href="https://www.npmjs.com/package/loopback4-ratelimiter">loopback4-ratelimiter</a></td>
+      <td>Provide rate limiting in LoopBack applications</td>
+    </tr>
+    <tr>
+      <td><a href="https://www.npmjs.com/package/loopback4-notifications">loopback4-notifications</a></td>
+      <td>Add different notification mechanisms vis-à-vis, Push, SMS, Email, to any LoopBack 4 based REST API application or microservice</td>
+    </tr>
+    <tr>
+      <td><a href="https://www.npmjs.com/package/loopback4-s3">loopback4-s3</a></td>
+      <td>Integrate AWS S3 in LoopBack applications</td>
+    </tr>
+    <tr>
+      <td><a href="https://www.npmjs.com/package/loopback-component-history">loopback-component-history</a></td>
+      <td>Soft create, edit, delete models, saving history of changes</td>
+    </tr>
+    <tr>
+      <td><a href="https://www.npmjs.com/package/loopback-component-filter">loopback-component-filter</a></td>
+      <td>Filter models in repository layer, using an async function</td>
+    </tr>
+  </tbody>
+</table>
+
+## Authentication and authorization related extensions
+
+<table>
+  <thead>
+    <tr>
+      <th width="160">Extension</th>
+      <th width="280">Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><a href="https://www.npmjs.com/package/loopback4-authentication">loopback4-authentication</a></td>
+      <td>Provide support for five passport based strategies</td>
+    </tr>
+    <tr>
+      <td><a href="https://www.npmjs.com/package/loopback4-authorization">loopback4-authorization</a></td>
+      <td>Implement authorization using simple string based permissions</td>
+    </tr>
+    <tr>
+      <td><a href="https://www.npmjs.com/package/loopback-component-crud">loopback-component-crud</a></td>
+      <td>Generic CRUD controller mixin, supports authentication, nested authorization(cascade)</td>
+    </tr>
+    <tr>
+      <td><a href="https://www.npmjs.com/package/loopback-component-authorization">loopback-component-authorization</a></td>
+      <td>Implement HRBAC authorization</td>
+    </tr>
+  </tbody>
+<table>
```

Comment:
Please add instructions on how can extension authors contribute their extensions to our list. Feel free to open a follow-up pull request for that.

You can inspire in our existing docs for contributing LB3 extensions: https://loopback.io/doc/en/community/How-to-add-a-project.html